docs/1.guide/4.event.md

```diff
@@ -13,28 +13,26 @@ An event is passed through all the lifecycle hooks and composable utils to use i
 **Example:**
 
 ```js
-import { defineEventHandler, readBody, query } from "h3";
-
-app.use(
-  defineEventHandler((event) => {
-    // Log event. `.toString()` stringifies to a simple string like `[GET] /<path>`
-    console.log(`Request: ${event.toString()}`);
-
-    // Parse query parms
-    const query = getQuery(event)
-
-    // Try to read request body
-    const body = await readBody(event).catch(() => {})
-
-    // Echo back request as response
-    return {
-      path: event.path,
-      method: event.method,
-      query,
-      body,
-    }
-  }),
-);
+import { defineEventHandler, getQuery, readBody } from "h3";
```

Comment:
This typo (`query` 👉 `getQuery`) had been already fixed in #658, before I recognized it.